stm32-modules/heater-shaker/firmware/motor_task/freertos_motor_task.cpp

```diff
@@ -79,8 +79,17 @@ void run(void *param) {
     memset(&_local_task.handles, 0, sizeof(_local_task.handles));
     _local_task.handles.plate_lock_complete = handle_plate_lock;
     motor_hardware_setup(&_local_task.handles);
-
     auto policy = MotorPolicy(&_local_task.handles);
+    auto &queue = _task.get_message_queue();
+    // ensure plate lock closed via message at startup (needed for homing)
+    auto message1 = messages::ClosePlateLockMessage{.from_startup = true};
+    static_cast<void>(
+        queue.try_send(message1, 1500));  // adjust ticks to ensure fully closed
```

Comment:
this time is the timeout before an error, it's not a delay before sending

Reply:
and since the task's not actually running at this time, no matter what number you put here the task is going to have these two messages immediately next to each other in the queue. so it will see closeplatelock, then home, on subsequent calls to `try_once`. this is fine if the `ClosePlateLock` message execution blocks until it's done, but otherwise it'll start homing before the close finishes.